vizro-core/examples/visual-vocabulary/pages/time.py

```diff
@@ -240,6 +240,46 @@
         ),
     ],
 )
+
+
+sparkline_page = vm.Page(
+    title="Sparkline",
+    path="time/sparkline",
+    layout=vm.Layout(grid=PAGE_GRID),
+    components=[
+        vm.Card(
+            text="""
+                #### What is a sparkline chart?
+
+                A sparkline chart is a compact line or area chart that displays multiple time series over a continuous
+                period. Without visible axes or labels, they are ideal for embedding within text, tables, or dashboards,
+                highlighting relative movement rather than precise values for a quick visual summary of trends.
+
+                &nbsp;
+
+                #### When should I use it?
+
+                Use sparkline charts to show trends for multiple time series sharing the same Y-axis quantity over the
+                same X-axis time range. They emphasize relative movement rather than precise values. To keep them
```

Comment:
```suggestion
                Use sparkline charts to show trends for multiple time series sharing the same y-axis quantity over the
                same x-axis time range. They emphasize relative movement rather than precise values. To keep them
```